TelCo.ColorCoder/StringPairs.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+ partial class Program
+    {
+        /// <summary>
+        /// Gives a big string containing the pair number corresponding to their major and minor colors
+        /// </summary>
+        /// <returns></returns>
+
+        private static void getAllPairsInString()
+        {
```

Comment:
getAllPairsInString needs to 'get' all pairs in a string and return it, not just output to console. Currently this 'get' function is getting nothing.